.github/workflows/scripts/get-codeowners.sh

```diff
@@ -20,9 +20,9 @@ get_codeowners() {
   #   ${1}: Insert first argument given to this function
   #   [\/]\?: Match 0 or 1 instances of a forward slash
   #   \s: Match any whitespace character
-  echo "$((grep -m 1 "^${1}[\/]\?\s" .github/CODEOWNERS || true) | \
+(grep -m 1 "^${1}[\/]\?\s" .github/CODEOWNERS || true) | \
         sed 's/   */ /g' | \
-        cut -f3- -d ' ')"
+        cut -f3- -d ' '
```

Comment:
To make sure I understand, the problem here is that the `echo` wasn't doing anything, is that right?

Reply:
absolutely